### DISCUSSION TITLE
flow.opera.com: breakage

### DISCUSSION THREAD

**Discussion by D4niloMR** [uBlockOrigin/uAssets#16864] [Q&A] [answered]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://flow.opera.com/ext/v1/index.html`

### Category

breakage

### Description

With [Hard Mode uBlock](https://github.com/gorhill/uBlock/wiki/Blocking-mode:-hard-mode) the Opera Flow connect and if refresh or close browser the session close, i tried add these filter allowing 3p request but don't work:
```
* opera.com * noop
* operacdn.com * noop
auth.opera.com * 3p noop
auth.opera.com operacdn.com * noop
auth.opera.com auth-statics.operacdn.com * noop
web.flow.opera.com * 3p noop
web.flow.opera.com flow.operacdn.com * noop
www.opera.com * 3p noop
www.opera.com operacdn.com * noop
flow.operacdn.com * * noop
flow.operacdn.com * 3p noop
```

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.46.0
Chromium: 108
filterset (summary): 
  network: 97199
  cosmetic: 186923
  scriptlet: 24958
  html: 0
listset (total-discarded, last-updated): 
  added: 
    adguard-annoyance: 64091-94, 3d.15h.52m
    adguard-social: 19845-831, 3d.15h.52m
    fanboy-annoyance: 84076-6436, 3d.15h.52m
    fanboy-cookiemonster: 45588-45588, 3d.15h.52m
    fanboy-social: 22774-22774, 3d.15h.52m
    fanboy-thirdparty_social: 68-20, 3d.15h.52m
    ublock-annoyances: 4832-211, 3d.15h.52m
  default: 
    user-filters: 24-1, never
    easylist: 61919-102, 3d.15h.52m
    easyprivacy: 30977-1261, 3d.15h.52m
    plowe-0: 3680-905, 3d.15h.52m
    spa-1: 5054-67, 3d.15h.52m
    ublock-abuse: 79-0, 3d.15h.52m
    ublock-badware: 4353-121, 3d.15h.52m
    ublock-filters: 32707-575, 3d.15h.52m
    ublock-privacy: 276-8, 3d.15h.52m
    ublock-quick-fixes: 175-17, 21h.21m
    ublock-unbreak: 1899-51, 3d.15h.52m
    urlhaus-1: 5908-0, 21h.20m
filterset (user): [array of 17 redacted]
switchRuleset: 
  added: [array of 1 redacted]
hostRuleset: 
  added: [array of 54 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  suspendUntilListsAreLoaded: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 389 ms (selfie)
  maxAssetCacheWait: 230 ms
popupPanel: 
  blocked: 0
```
</details>

**Comment by iam-py-test** [comment, marked as answer]
Can you try adding this to your "my rules" tab and retesting:
```
flow.opera.com b.akamaiedge.net * noop
```
This seems to work fine in Firefox, but it might be different in Opera/Chrome
If that doesn't fix it:
- try disabling your custom filterlists
> ```yml
>    adguard-annoyance: 64091-94, 3d.15h.52m
>    adguard-social: 19845-831, 3d.15h.52m
>    fanboy-annoyance: 84076-6436, 3d.15h.52m
>    fanboy-cookiemonster: 45588-45588, 3d.15h.52m
>    fanboy-social: 22774-22774, 3d.15h.52m
>    fanboy-thirdparty_social: 68-20, 3d.15h.52m
>    ublock-annoyances: 4832-211, 3d.15h.52m
> ```
- Open the logger, reproduce the issue, and either try to figure out what is causing the issue yourself or post back here

**Reply by D4niloMR** [reply]
Worked, thank you

**Comment by gorhill** [comment]
[Blocking mode: hard mode](https://github.com/gorhill/uBlock/wiki/Blocking-mode:-hard-mode):

> Very high likelihood of web pages being broken: **you have to be ready and willing to fix them when this happen.**

Please do not open self-inflicted issues as a result of your usage of advanced blocking modes when the issue is not caused by a filter in a filter lists. Volunteers here are already burdened enough with maintaining filter lists. You had all the required information in the popup panel and logger to figure out on your own what your hard mode rules were blocking and causing the breakage.

**Reply by D4niloMR** [reply]
Ok, sorry for that.
